src/Template/Imagemap/ImagemapTemplate.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace LineMob\Core\Template\Imagemap;
+
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapMessageActionBuilder;
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapUriActionBuilder;
+use LINE\LINEBot\MessageBuilder\Imagemap\BaseSizeBuilder;
+use LINE\LINEBot\MessageBuilder\ImagemapMessageBuilder;
+use LineMob\Core\Template\AbstractTemplate;
+
+class ImagemapTemplate extends AbstractTemplate
+{
+    /**
+     * @var string
+     */
+    protected $baseUrl;
+
+    /**
+     * @var string
+     */
+    protected $altText = 'this is an imagemap';
+
+    /**
+     * @var BaseSize
+     */
+    protected $baseSize;
+
+    /**
+     * @var ImagemapAction[]
+     */
+    protected $actions;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplate()
+    {
+        $baseUrl = $this->baseUrl;
+        $altText = $this->altText;
+
+        $baseSizeHeight = $this->baseSize->height;
+        $baseSizeWidth = $this->baseSize->width;
+        $baseSizeBuilder = new BaseSizeBuilder($baseSizeHeight, $baseSizeWidth);
```

Comment:
@liverbool 
Because i think `ImagemapMessageBuilder` has 4 parameters if we use only `$this->property` like this it will take a long word So I use **var** for short parameters.